wazuh/wazuh
# Add the processing of vulnerabilities without operation

This PR solves the issue described in https://github.com/wazuh/wazuh/issues/3277.

### How to test it

1) Install a package with a non-fixed vulnerability, like `fdisk`.

2) Run a vulnerability scan. If you are using Ubuntu 18:

``` XML
<feed name="ubuntu-18">
    <disabled>no</disabled>
    <update_interval>1h</update_interval>
</feed>
```

3) Now you will see alerts so far unreported in this system. If you have installed `fdisk`, you should be vulnerable (according to Ubuntu) to `CVE-2018-7738`.

### Packages lightly related to the vulnerability

Testing this PR we have noticed massive flooding of alerts in Ubuntu 18. This is due to a change in the syntax of the comment that tells us that we have to discard some packages from a vulnerability.

For example, the `CVE-2015-8548` for `libv8` in the Ubuntu Precise OVAL is pending to evaluation, while in Ubuntu Bionic has unlinked the package from the vulnerability. 

- **Precise note**: `The vulnerability of the 'libv8' package in precise is not known (status: 'needs-triage'). It is pending evaluation.`
- **Bionic note:** `libv8-3.14.5: while related to the CVE in some way, a decision has been made to ignore this issue (note: 'libv8 not supported').`

Unfortunately, there is no other way to check these conditions than to analyse this comment.
